homeassistant/components/hassio/discovery.py

```diff
@@ -17,7 +17,7 @@
 
 
 @callback
-def async_setup_discovery_view(hass: HomeAssistantView, hassio):
+def async_setup_discovery_view(hass: HomeAssistant, hassio):
```

Comment:
We should not add partial typing, this should be:
`def async_setup_discovery_view(hass: HomeAssistant, hassio: HassIO) -> None:`

Reply:
True, but please note that this PR did not _add_ partial typing, it just corrected the wrong type in the existing partial one.